pkg/daemon/update.go

```diff
@@ -1073,6 +1073,7 @@ func getSupportedExtensions() map[string][]string {
 	// These are RHCOS supported extensions.
 	// Each extension keeps a list of packages required to get enabled on host.
 	return map[string][]string{
+		"crun-wasm":            {"crun-wasm"},
```

Comment:
I think it's worth thinking about this name. For instance, `sandboxed-containers` is name for installing `kata`. should we name this `wasm` or `wasm-container-runtime` or something?

Reply:
Yeah, I will stick to `wasm`.